pages/projects/_projectId.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="project-details">
+    <div class="project-details__page-route">
```

Comment:
This should be a generic breadcrumb component, as it will be on every detail page.